app/src/main/java/app/revanced/integrations/settingsmenu/ReVancedSettingsFragment.java

```diff
@@ -126,6 +163,33 @@ public void onCreate(Bundle bundle) {
             sharedPreferences.registerOnSharedPreferenceChangeListener(this.listener);
             this.Registered = true;
 
+            this.screens = new ArrayList<>();
+            this.screens.add((PreferenceScreen) getPreferenceScreen().findPreference("ads"));
+            this.screens.add((PreferenceScreen) getPreferenceScreen().findPreference("interactions"));
+            this.screens.add((PreferenceScreen) getPreferenceScreen().findPreference("layout"));
+            this.screens.add((PreferenceScreen) getPreferenceScreen().findPreference("misc"));
+            this.screens.add((PreferenceScreen) getPreferenceScreen().findPreference("revanced_pref_default_video_quality"));
```

Comment:
Code such as this could fail if all ads or interaction patches are excluded for example. All in all this code would need proper refactoring before getting merged.